pkgs/top-level/release-staging.nix

```diff
@@ -0,0 +1,71 @@
+/*
+A release file that builds few packages.
+
+The aim is to test the `staging` branch before it is merged into `staging-next` at which points more builds are run.
+
+*/
+
+
+{ nixpkgs ? { outPath = (import ../../lib).cleanSource ../..; revCount = 1234; shortRev = "abcdef"; }
+, supportedSystems ? [ "x86_64-linux" "x86_64-darwin" ]
+}:
+
+with import ./release-lib.nix { inherit supportedSystems; };
+
+
+(import ./release-small.nix {inherit nixpkgs; inherit supportedSystems;}) // (mapTestOn (rec {
+
+  # Some tools
+  diffoscope = all;
+  emacs = all;
+  git = all;
+  nox = all;
+  vim = all;
+
+  llvm = all;
+
+  linuxPackages.kernel = linux;
+
+  ## Package sets - Here a certain package is chosen simply to test the package set logic
+
```

Comment:
I think binaries like these should be included, unless they are already.

1. `coreutils`
2. `binutils`